docs/examples/vector_stores/SupabaseVectorIndexDemo.ipynb

```diff
@@ -0,0 +1,230 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "db0855d0",
+   "metadata": {},
+   "source": [
+    "# Supabase Vector Store\n",
+    "In this notebook we are going to show how to use [Vecs](https://supabase.github.io/vecs/) to perform vector searches in LlamaIndex"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c2d1c538",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "import sys\n",
+    "\n",
+    "# Uncomment for debugging the local copy of Llamaindex you are working on\n",
+    "# sys.path.append('[dir_to_llama_index_for_testing]')\n",
+    "\n",
+    "# Uncomment to see debug logs\n",
+    "# logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)\n",
+    "# logging.getLogger().addHandler(logging.StreamHandler(stream=sys.stdout))\n",
+    "\n",
+    "from llama_index import SimpleDirectoryReader, Document, StorageContext\n",
+    "from llama_index.indices.vector_store import GPTVectorStoreIndex\n",
+    "from llama_index.vector_stores import SupabaseVectorStore\n",
+    "import textwrap"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26c71b6d",
+   "metadata": {},
+   "source": [
+    "### Setup OpenAI\n",
+    "The first step is to configure the openai key. It will be used to created embeddings for the documents loaded into the index"
```

Comment:
```suggestion
    "The first step is to configure the OpenAI key. It will be used to created embeddings for the documents loaded into the index"
```